spec/kibana_spec.rb

```diff
@@ -1,24 +1,9 @@
 # encoding: utf-8
 require 'spec_helper'
 
-kibana_version = '4.4.1'
```

Comment:
Cribbing an example from the logstash spec:

```
$ grep 'describe package' spec/logstash_spec.rb  -A 5
describe package('logstash') do
  it { should be_installed }
  # Might be a little aggressive to test for exact version number.
  # Seeing an oddly formatted version number; ElasticSearch is simply '2.1.1'.
  its('version') { should >= '1:2.1.2-1' }
end
```

We can use the same syntax for the kibana checks. I'll add it!